cloudman/values.yaml

```
@@ -213,20 +203,26 @@ helmsman_config:
               storageClass: nfs
               accessMode: ["ReadWriteMany"]
               size: 10Gi
-
-          extraVolumes:
-            - name: project-data
-              persistentVolumeClaim:
-                claimName: "{{context.project.namespace}}-project-data-pvc"
-            - name: public-data
-              persistentVolumeClaim:
-                claimName: "{{context.project.namespace}}-public-data-pvc"
-
-          extraVolumeMounts:
-            - name: project-data
-              mountPath: /gvl/projects/current
-            - name: public-data
-              mountPath: /gvl/public
+            extraVolumes:
+              - name: project-data
+                persistentVolumeClaim:
+                  claimName: "{{context.project.namespace}}-project-data-pvc"
+              - name: public-data
+                persistentVolumeClaim:
+                  claimName: "{{context.project.namespace}}-public-data-pvc"
+              - name: notebook-conf
+                configMap:
+                  name: {{context.domain}}-projman-configs
```

Since context.domain could be an ip address, it may not always be a valid configmap name. Also, this configmap will be uninstalled along with the project right?
```suggestion
                  name: {{context.project.namespace}}-projman-configs
```

---

Yes, that was just wrong, went on bad autopilot heh